storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst

```diff
@@ -185,6 +185,18 @@ option, which will copy the backup to the server's :term:`datadir`:
 If you don't want to save your backup, you can use the :option:`xtrabackup
 --move-back` option which will move the backed up data to the :term:`datadir`.
 
+.. note::
+
+    When ``relay-log-info-repository=TABLE`` is enabled, the instance recovered from the backup has errors in the error log, like the following:
+    
+    .. code-block:: bash
+    
+        2019-08-09 12:40:02 69297 [ERROR] Failed to open the relay log '/data/mysql-relay-bin.004349' (relay_log_pos 5534092)
+    
+    To avoid these types of issues, enable ``relay_log_recovery``.
```

Comment:
Also add:

Or execute `RESET SLAVE` prior to `CHANGE MASTER TO`

Reply:
Added and moved.